### ISSUE TITLE
#407: added basicauth docs, reorganized menu

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#422]
_From @uweschaefer on 2019-05-20 08:08:55 +0000 UTC_

_Copied from original issue: Mercateo/factcast#422_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2019-05-20 08:11:33 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/422?src=pr&el=h1) Report
> Merging [#422](https://codecov.io/gh/Mercateo/factcast/pull/422?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/83b1a7ba7cd81a1a78dc71f78b338a18499bbc4d?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.


```diff
@@           Coverage Diff           @@
##           master     #422   +/-   ##
=======================================
  Coverage   88.87%   88.87%           
=======================================
  Files          78       78           
  Lines        1852     1852           
  Branches      168      168           
=======================================
  Hits         1646     1646           
  Misses        126      126           
  Partials       80       80
```

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2019-05-20 10:47:05 +0000 UTC_

resolves #407
